ddtrace/tracer/option.go

```diff
@@ -443,6 +443,13 @@ func WithHostname(name string) StartOption {
 	}
 }
 
+// WithLogStartup allows control over whether to print the startup log or not
+func WithLogStartup(logStartup bool) StartOption {
```

Comment:
Except maybe let's rename the argument to something else.
```suggestion
func WithLogStartup(yes bool) StartOption {
```

Reply:
I'm going with `enabled` to be consistent with our other boolean `StartOption`s.